pkg/interactive/interactive.go

```diff
@@ -38,6 +40,30 @@ type Input struct {
 	Validators []Validator
 }
 
+func (i *Input) LenOptions() int {
```

Comment:
Not sure i understand why this method is needed instead of using `len(input.Options)` directly?